Update delete-resource-group.md

azarboon opened the pull request:
The following should be clearly stated in the doc because currently this information is scattered:

"To delete a resource group, firstly you need to delete its underlying resource locks as well as backup data."

azarboon committed:
Update delete-resource-group.md

The following should be clearly stated in the doc because currently this information is scattered:

"To delete a resource group, firstly you need to delete its underlying resource locks as well as backup data."

mumian commented on articles/azure-resource-manager/management/delete-resource-group.md:
```diff
@@ -13,6 +13,9 @@ ai-usage: ai-assisted
 This article shows how to delete resource groups and resources. It describes how Azure Resource Manager orders the deletion of resources when you delete a resource group.
 
+> [!NOTE]  
+> To delete a resource group, firstly you need to delete its underlying resource locks as well as backup data.
```
Can you add a link to the resource locks?  For example:

To delete a resource group, you must first remove any underlying [resource locks](./lock-resources.md) and backup data.

azarboon replied:
Looks good. Let me try to change this.

azarboon reviewed (commented):
Approved

azarboon committed:
Update delete-resource-group.md

Followed the feedback and added link to the text.

azarboon committed:
Update delete-resource-group.md

Changed the link to https://github.com/MicrosoftDocs/azure-docs/pull/delete-resource-group.md

azarboon committed:
Update delete-resource-group.md

Removed the link. Kindly please sign off  this PR and afterwards, add the link.
